java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -734,7 +728,13 @@ class ReferenceModeStore private constructor(
                 containerStore,
                 storageKey.backingKey,
                 type.containedType
-            )
+            ).also { refModeStore ->
+                containerStore.on(ProxyCallback {
```

Comment:
maybe a comment on why this is here now. If only we could have suspending constructors...

Reply:
Good idea. Done, and added a note in commit message as well.